packages/arb-token-bridge-ui/tests/e2e/specs/readClassicDeposits.cy.ts

```diff
@@ -1,4 +1,4 @@
-import { Transaction, AssetType } from 'token-bridge-sdk'
+import { Transaction, AssetType } from '../../../src/token-bridge-sdk'
```

Comment:
`token-bridge-sdk` alias import was not getting correctly resolved in e2e/tests. Probably because of multiple `tsconfig.json` at each level. Tried redefining the alias in different tsconfigs but couldn't catch a break.